promql/promqltest/testdata/functions.test

```diff
@@ -115,11 +109,10 @@ load 5m
 
 # Counter resets at in the middle of range are handled correctly by rate().
 eval instant at 50m rate(testcounter_reset_middle[50m])
-	{} 0.03
+	{} 0.02962962962962963
```

Comment:
Here we would also like to see a round result. Does it work if you extend the range to 55m?

Reply:
Again, we need to change a few more things. Here my suggestion:

```Go
# Tests for rate().
load 5m
	testcounter_reset_middle	10+10x4 0+10x5
	testcounter_reset_end    	10+10x9 0 10

# Counter resets at in the middle of range are handled correctly by rate().
eval instant at 50m rate(testcounter_reset_middle[55m])
	{} 0.03
```